docs/ossm/ossm2-migration/network-policies.md

```diff
@@ -0,0 +1,303 @@
+    So if you have followed the other steps in the checklist, this will not exist.
+- **Location**: Created in SMCP namespace
+- **Sample YAML**:
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: istio-ingressgateway             # Gateway name
+  namespace: istio-system               
+  labels:
+    maistra-version: "2.6.5"
+    app: istio-ingressgateway            # Gateway app label
+    istio: ingressgateway                # Gateway component label
+  annotations:
+    "maistra.io/internal": "true"
+spec:
+  podSelector:                           # Targets ingress gateway pods
+    matchLabels:
+      app: istio-ingressgateway
+      istio: ingressgateway
+  ingress:
+    - {}                                 # Empty rule allows all ingress traffic
+```
+
+## How to Migrate Network Policies to 3.0
+
+### Migration Steps
+
+#### For SMCP Namespace
+Recreate necessary Network Policies in the new Service Mesh 3.0 control plane namespace:
+- Istiod Network Policy
+- Default Mesh Network Policy
+- Expose Route Network Policy
+- Ingress Gateway Network Policy (if you were previously using SMCP-created gateways)
+
+#### For Workload Namespaces
+For each namespace that was part of the 2.6 mesh:
+
+1. Recreate the following Network Policies:
+    - Default Mesh Network Policy
+    - Expose Route Network Policy
+
+2. Update labels:
+    - Consider replacing the `maistra.io/expose-route: "true"` label with a new label scheme
+    - Update corresponding Network Policy selectors to match new labels
+
+### Important Notes
+- Simply removing ownerReferences from existing Network Policies won't prevent their deletion during migration.
+- When ServiceMeshMember is removed from a namespace, the `maistra.io/member-of` label is automatically removed from the namespace.
+- Duplicate NetworkPolicies in the same namespace should not cause issues.
+
+### Best Practices
+1. Test Network Policies in a non-production environment first.
+2. If possible, create Network Policies after migrating workloads to ensure continuous protection. If it is not possible due to security policies to migrate workloads without Network Policies, see below for tips.
+
+### Example Network Policies with both 2.6 and 3.0 present
+
+Some users may not be able to go without Network Policies during the migration process due to security concerns. 
+This can be tricky, as during the migration both control planes must have access to all workloads and vice versa. Created Network Policies must not block traffic for either control plane.
+
+To assist users in this situation, here is an example scenario with example Network Policies. 
+
+A user has a 2.6 control plane with the name `default` in the namespace `istio-system` and workloads in namespaces `httpbin-2` and `httbin-3`. They need to ensure Network Policies remain active to fulfill security obligations. 
```

Comment:
The same note about `default` name as above

Reply:
renamed to `basic` here [9a057b6](https://github.com/openshift-service-mesh/sail-operator/pull/203/commits/9a057b69469175286ebf1f9721b504264decf68a)